hexrd/crystallography.py

```diff
@@ -816,6 +816,73 @@ def set_exclusions(self, exclusions):
         return
     exclusions = property(get_exclusions, set_exclusions, None)
 
+    def exclude(
+            self, dmin=None, dmax=None, tthmin=None, tthmax=None,
+            sfacmin=None, sfacmax=None, pintmin=None, pintmax=None
+    ):
```

Comment:
I think it would be valuable if we modified your test script for this function and put it in the [`tests`](https://github.com/HEXRD/hexrd/tree/master/tests) so that it runs with the CI.

In general, I think having more tests running in the HEXRD CI would be very helpful for future development.